index.bs

```diff
@@ -655,11 +659,15 @@ urlPrefix: https://w3c.github.io/FileAPI/#dfn-; type: dfn;
 
 			1. Let |p| be a new [=Promise=].
 
+			1. If the [=current settings object's=] responsible document is not
```

Comment:
Same as above... immediately return the rejected promise, otherwise continue in parallel.